deploy/bundle/metadata/annotations.yaml

```diff
@@ -4,7 +4,7 @@ annotations:
   operators.operatorframework.io.bundle.manifests.v1: manifests/
   operators.operatorframework.io.bundle.metadata.v1: metadata/
   operators.operatorframework.io.bundle.package.v1: devworkspace-operator
-  operators.operatorframework.io.bundle.channels.v1: fast
+  operators.operatorframework.io.bundle.channels.v1: alpha
```

Comment:
Looks like fast got switched back to alpha

Reply:
yeah, it should be done by adding argument to operator-sdk but not by modifying files.